components/brave_rewards/resources/ui/components/pageWallet.tsx

```diff
@@ -156,9 +156,14 @@ class PageWallet extends React.Component<Props, State> {
     }
 
     return grants.map((grant: Rewards.Grant) => {
+      let expireDate = ''
```

Comment:
nit: same logic is used in a couple places, consider decoupling into a method?

Reply:
we only use it here for the settings page, sadly extension code has different util files